ibis/sql/tests/test_udf.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2015 Cloudera Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ibis.compat import unittest
+import ibis.sql.udf as udf
+from ibis.sql.exprs import _operation_registry
+from ibis.expr.operations import ValueOp
+
+
+class UDFTest(unittest.TestCase):
+
```

Comment:
We need tests that validate each kind of input type and output type

1) more than one input parameter (preferably covering each Ibis type -- 4 integer types, float, double, boolean, timestamp)

2) each return type